src/github.com/getlantern/lantern-mobile/app/src/main/java/org/getlantern/lantern/fragment/FeedFragment.java

```diff
@@ -31,29 +31,22 @@
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        this.mFeedItems = new ArrayList<FeedItem>();
+        this.adapter = new FeedAdapter(getActivity(), mFeedItems);
 
         Bundle bundle = getArguments();
-        this.feedName = bundle.getString("name");
-        this.mFeedItems = Collections.synchronizedList(new ArrayList<FeedItem>());
+        if (bundle != null) {
+            this.feedName = bundle.getString("name");
```

Comment:
Whenever the feed fragment is created, we're always passing in a bundle with a name argument. There's also multiple checks before that to make sure we have a valid name. That said, I'm returning an empty list of feed items now whenever we encounter a null or empty string name. I also removed setFeedName (wasn't being used anyway).